cli-subcommands/github-handler.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-var GithubHandler = require('../lib/GithubHandler');
+var GitHubHandler = require('../lib/GitHubHandler');
```

Comment:
This change requires that Githubhandler.js be renamed which it currently isn't and will fail on a case-sensitive file system. I have a patch to go into this as soon as I get the open source stack running on Node 12.

Reply:
Looks renamed in the PR, but when analyzing the feature/node-12 branch it is GithubHandler instead of GitHubHandler. We should verify.